docs/UserGuide.md

```diff
@@ -370,7 +373,7 @@ request that does not specify any data or header **(an ongoing call can be cance
 
 <div markdown="span" class="alert alert-warning">:bulb: **Tip:**
 The above shortcut for run command is designed for users to easily verify outputs for common API endpoints that 
-do not require other input data or header. Note that this feature is only meant for sending GET requests.
+do not require any input data or header. Note that this feature is only meant for sending GET requests.
```

Comment:
do -> does?

Reply:
er, ... endpoints do not ...., my grammar is correct, right?

Reply:
yep, did not see the previous line oopps

Reply:
no prob